apis/facebook/api/src/objects/fb_user.e

```diff
@@ -0,0 +1,223 @@
+note
+	description: "Object representing a Facebook User."
+	date: "$Date$"
+	revision: "$Revision$"
+	EIS: "name=Facebook User", "src=https://developers.facebook.com/docs/graph-api/reference/user/", "protocol=uri"
+
+class
+	FB_USER
+
+feature -- Access: Basic Properties
+
+	id:  detachable STRING
+			-- The id of this person's user account.
+
+	name: detachable STRING
+			-- The person's full name.
+			--! Predeterminded.
+
+	birthday: detachable STRING
+			-- The person's birthday.
+
+	email: detachable STRING
+			--	The person's email.
+
+	first_name: detachable STRING
+			-- The person's first name.
+
+	last_name: detachable STRING
+			--	The person's LAST name.			
+
+	gender: detachable STRING
+			--  The gender selected by this person, male or female.
+
+	middle_name: detachable STRING
+			-- The person's middle name.
+
+	time_zone: REAL
+			-- 	timezone float (min: -24) (max: 24)	
+
+	locale: detachable STRING
+			-- The person's locale
+
+	link: detachable STRING
+			-- A link to the person's Timeline
+
+	location: detachable FB_PAGE
```

Comment:
I've double check and the FB doc define it as a PAGE, https://developers.facebook.com/docs/graph-api/reference/user/